### PR TITLE
Feat/local setup

### PR DESCRIPTION
**Pull request by rodentskie**
**Story/Ticket Reference ID from JIRA:**
[JIRA](https://intersective.atlassian.net/browse/DEV-358)

**Additional Comments:**
`local` setup using docker

**Checklist:**

- [x] Unit test completed?: (Y/N)
- [x] Docs folder up to date?: (Y/N)
- [x] Add if anything missed: (Y/N)

All checklist are okay.

This PR looks great - it's ready to merge! Please review and let's ship it. :)

### CODE REVIEW

File: .github/workflows/p2-stage.yml
Comment (Sunilsbcloud) [inline .github/workflows/p2-stage.yml:1]: @rodentskie -> it should change back to trunk branch?

File: src/index.html
Comment (Sunilsbcloud) [inline src/index.html:1]: @rodentskie -> This segment changes required?

**Review by Sunilsbcloud — COMMENTED** [review]
Added comments to appropriate files

File: src/index.html
Comment (rodentskie) [reply src/index.html:1]: Yes, it is already there.

**Review by Sunilsbcloud — APPROVED** [review]

**Pull request merged**